src/zoauthServer.js

```diff
@@ -7,9 +7,10 @@
 import { StringTools, Password } from "zoapp-core";
 import createModel from "./model";
 import Route from "./model/route";
+import { constants } from "./constants";
 
-const GRANT_TYPE_PASSWORD = "password";
-const GRANT_TYPE_REFRESH_TOKEN = "refresh_token";
+const GRANT_TYPE_PASSWORD = constants.grant_type.password;
+const GRANT_TYPE_REFRESH_TOKEN = constants.grant_type.refresh_token;
```

Comment:
I dont think that's what @mikbry asked. Your constants were OK, just put them as is in the constants.js file.

Reply:
Like this ? 
```
export const constants = {
  GRANT_TYPE_PASSWORD: "password",
  GRANT_TYPE_REFRESH_TOKEN: "refresh_token",
};
```

Reply:
no, lik:

```
export const GRANT_TYPE_PASSWORD = "password";
```